pages/index.js

```diff
@@ -193,9 +193,16 @@ function HomePage() {
         </Paragraph>
       </Container>
 
-      <Container className="text-center pb-10" wide>
+      <Container className="text-center py-10" wide>
+      <Heading 
+          level="h3"
+          typeStyle="heading-lg"
+          className="mb-4"
+        >
+          Testimonials
```

Comment:
Can we come up with another title? What about something like "What the experts say"?